operator/v1/types_ingress.go

```diff
@@ -501,6 +535,12 @@ type AccessLogging struct {
 	//
 	// +optional
 	HttpLogFormat string `json:"httpLogFormat,omitempty"`
+
+	// httpCaptureHeaders defines HTTP headers that should be captured in
+	// access logs.  If this field is empty, no headers are captured.
+	//
+	// +optional
+	HTTPCaptureHeaders *IngressControllerCaptureHTTPHeaders `json:"httpCaptureHeaders,omitempty"`
```

Comment:
This doesn't need to be a pointer, because being present without any rules means the same thing, right?  We've avoided pointers in config

Reply:
You're right, that makes sense.

Reply:
You're right, that makes sense.